test/units/parsing/fixtures/ajson.json

```diff
@@ -6,7 +6,7 @@
         "baz": [
             {
                 "password": {
-                    "__ansible_vault": "$ANSIBLE_VAULT;1.1;AES256\n34646264306632313333393636316562356435376162633631326264383934326565333633366238\n3863373264326461623132613931346165636465346337310a326434313830316337393263616439\n64653937313463396366633861363266633465663730303633323534363331316164623237363831\n3536333561393238370a313330316263373938326162386433313336613532653538376662306435\n3339\n"
+                    "__ansible_vault": "$ANSIBLE_VAULT;1.1;AES256\n34646264306632313333393636316562356435376162633631326264383934326565333633366238\n3863373264326461623132613931346165636465346337310a326434313830316337393263616439\n64653937313463396366633861363266633465663730303633323534363331316164623237363831\n3536333561393238370a313330316263373938326162386433313336613532653538376662306435\n3338\n"
```

Comment:
Why did you change this?

Reply:
because all instances have the same data. I think it does not make sense to pass the same to the test multiple times

Reply:
Instances of what?

Reply:
in test/units/parsing/fixtures/ajson.json, all __ansible_vault values are the same